src/player.cpp

```diff
@@ -1,228 +1,278 @@
 #include "include\player.h"
 #include <string>
 
-std::string Player::getName() const
+Player::Player(const Player &player)
```

Comment:
Create a constructor that takes "(name, seed)" as it's parameters, then use this for player creation from the data input a the start, and for the ringer

Reply:
Done